vm/Makefile

```diff
@@ -34,5 +34,9 @@ libubpf.a: ubpf_vm.o ubpf_jit_x86_64.o ubpf_loader.o
 
 test: test.o libubpf.a
 
+install:
+	install -m 644 libubpf.a /usr/local/lib
+	install -m 644 inc/ubpf.h /usr/local/include
```

Comment:
usual convention is to have DESTDIR=/usr/local at top of Makefile to allow override installation directory

Reply:
Good point @ygrek. I will respin.